src/plone/api/tests/test_content.py

```diff
@@ -345,6 +345,43 @@ def unregister_indexer():
         # check that the exception is the one we raised
         self.assertEqual(ude.exception.reason, unicode_exception_message)
 
+    def test_create_at_with_title_in_request(self):
+        """ Test that content gets created with the correct title, even if
+            request.form['title'] already exists and has a different value.
+            This can occur, for example, when adding a Plone with an enabled
+            product that creates a site structure. In that case, the 'title'
+            would be that of the portal.
+            Only AT content types are affected, due to content.processForm.
+        """
+        leaked_title = 'This should not be set on content items'
+        self.layer['request'].form['title'] = leaked_title
+
+        container = self.portal
+
+        # Create a folder
+        folder = api.content.create(
+            container=container,
+            type='Folder',
+            id='test-folder',
+            title='Test folder'
+        )
+        assert folder
```

Comment:
just being curious: what's the point on this assertions?

Reply:
@hvelarde thank you for your feedback! I copied the test from `test_create_content` and forgot those. I'll clean them up right away.